script/docker_push

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
```

Comment:
Does this not result in the password being visible in the Travis log?

Reply:
you should encrypt them via https://docs.travis-ci.com/user/environment-variables#encrypting-environment-variables